pkg/wire/payload/transaction/base.go

```diff
@@ -213,3 +210,23 @@ func (b *Base) TXOs() []*Input {
 func (b *Base) Witness() []*Witness {
 	return b.Witnesses
 }
+
+// TypeTx returns the Type in the tx
```

Comment:
We can get rid of these methods for Type, Version, etc because they are exported on the baseTx. To get them we can call BaseTx().Version